cpp/ql/test/query-tests/Security/CWE/CWE-190/semmle/TaintedAllocationSize/test.cpp

```diff
@@ -180,6 +180,15 @@ void more_bounded_tests() {
 		}
 	}
 
+	{
+		int size = atoi(getenv("USER"));
+
+		if (size % 100)
+		{
+			malloc(size * sizeof(int)); // GOOD
```

Comment:
Why is this example "GOOD", but the one from lines 132-136 isn't?
That is, I can't see a real difference here:
https://godbolt.org/z/bPWaoPMYY

Reply:
I guess my actual question is: how is modulo supposed to bound the input here?

Reply:
Thanks for pointing this out @intrigus-lgtm. It's fixed now by #17269.